### PR TITLE
Theme Change Handler Added In Meetings Sidepanel 

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

File: samples/meeting-tabs/nodejs/src/components/shareview.jsx [hunk 1]
```diff
@@ -4,13 +4,37 @@
...
+                    break;
+                case 'default':
+                    this.setState({ appTheme: "timerCount-light" })
+                    break;
+                default:
+                    return this.setState({ appTheme: "timerCount-dark" })
+            }
+        });
+
+        // counts times
```
Comment (Prithvi-MSFT) [inline samples/meeting-tabs/nodejs/src/components/shareview.jsx:37]: fix typo

Comment (Mohammed-MSFT) [reply samples/meeting-tabs/nodejs/src/components/shareview.jsx:37]: Comment fixed

**Comment by Pawank-MSFT** [comment]
Executed buildpipeline: 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6733782950

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Mohammed-MSFT** [comment]
**Build pipeline executed successfully**
image

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Pawank-MSFT** [comment]
Executed build pipeline
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6994082886

**Pull request merged**